tldw_Server_API/tests/RAG_NEW/unit/test_unified_pipeline_focused.py

```
@@ -432,6 +434,153 @@ async def run(self, **kwargs):
+            def __init__(self, _analyze):
+                self.verifier = types.SimpleNamespace(verify=self._verify)
+
+            async def _verify(self, **kwargs):
+                verify_calls.append(kwargs)
+                return StubVerification(kwargs["claim"])
+
+            async def run(self, **kwargs):
+                raise AssertionError("engine.run should not execute when pre-extracted claims exist")
+
+        class StubClaim:
+            def __init__(self, id, text, span=None):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)